### PR TITLE
Create sysmon_cve_2021_31979_cve_2021_33771_exploits.yml

### PR DESCRIPTION
**Pull request by BlackB0lt**

### CODE REVIEW

**Comment by BlackB0lt** [comment]
@Neo23x0 Could you help to resolve my error in test-sigma?

File: rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+title: CVE-2021-31979 CVE-2021-33771 Exploits by Sourgum
```
Comment (frack113) [inline rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml:1]:
```
action: global
title: CVE-2021-31979 CVE-2021-33771 Exploits by Sourgum
```

File: rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+detection:
+    selection:
+        EventID: 13
+        TargetObject|contains: 
+            - '\Software\Classes\CLSID\{CF4CC405-E2C5-4DDD-B3CE-5E7582D8C9FA}\InprocServer32'
+            - '\SOFTWARE\Classes\CLSID\{7C857801-7381-11CF-884D-00AA004B2E24}\InProcServer32'
+    keywords:
+        - IMJPUEXP.DLL
+    condition: selection and keywords
+falsepositives:
```
Comment (frack113) [inline rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml:47]: As falsepositives and level are commun (global) for all detection move before the first `---`

File: rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            - 'C:\Windows\system32\ime\IMEJP\WimBootConfigurations.ini'
+            - 'C:\Windows\system32\ime\IMETC\WimBootConfigurations.ini'
+    condition: selection
+---
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+        EventID: 13
```
Comment (frack113) [inline rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml:40]: For sysmon don't Eventid use `category: registry_event`

File: rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - https://www.microsoft.com/security/blog/2021/07/15/protecting-customers-from-a-private-sector-offensive-actor-using-0-day-exploits-and-devilstongue-malware/
+    - https://citizenlab.ca/2021/07/hooking-candiru-another-mercenary-spyware-vendor-comes-into-focus/
+tags:
+    - attack.credential_access
+    - attack.t1566
+    - attack.t1203
+    - cve.2021-33771
+    - cve.2021-31979
+    - threat_group.Sourgum
+logsource:
```
Comment (frack113) [inline rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml:17]:
```
---
logsource:
```

**Review by frack113 — CHANGES_REQUESTED** [review]
Hi, some change to try again

File: rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            - 'C:\Windows\system32\config\cy-GB\Setup\SKB\InputMethod\TupTask.dat'
+            - 'C:\Windows\system32\config\config\startwus.dat'
+            - 'C:\Windows\system32\ime\SHARED\WimBootConfigurations.ini'
+            - 'C:\Windows\system32\ime\IMEJP\WimBootConfigurations.ini'
+            - 'C:\Windows\system32\ime\IMETC\WimBootConfigurations.ini'
+    condition: selection
+---
+logsource:
+    product: windows
+    service: sysmon
```
Comment (frack113) [inline rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml:42]: Please remove `service` as it is not need.
When you convert `registry_event` and `windows` it give sysmon chanel and the good eventid in the query.
Thanks

Comment (BlackB0lt) [reply rules/windows/sysmon/sysmon_cve_2021_31979_cve_2021_33771_exploits.yml]: Done

**Review by frack113 — APPROVED** [review]

**Pull request merged**